src/CPUSolver.cpp

```diff
@@ -1590,6 +1560,8 @@ double CPUSolver::computeResidual(residualType res_type) {
     FP_PRECISION* nu_sigma_f;
     Material* material;
 
+#pragma omp parallel for private(new_fission_source, old_fission_source,\
+     material, nu_sigma_f) schedule(static)
```

Comment:
That would work as well. The style guide probably says how we should do it but i haven't looked.
The loop wasn't parallelized before. If it had been without the  private clause it would have been completely broken.